lib/archethic/mining/pending_transaction_validation.ex

```diff
@@ -139,6 +141,18 @@ defmodule Archethic.Mining.PendingTransactionValidation do
     end
   end
 
+  defp valid_origin_signature(tx = %Transaction{}) do
```

Comment:
Question: I don't get why this check is done in pending transaction validation. The purpose of the proof of work is to be done by coordinator mainly. The case where the proof of work is done by cross validation nodes is when the proof of work is empty. Maybe the logic should be the same